Sources/TokamakCore/Shapes/Path.swift

```diff
@@ -25,6 +25,17 @@ import WASILibc
 
 /// The outline of a 2D shape.
 public struct Path: Equatable, LosslessStringConvertible {
+  public class PathBox: Equatable {
```

Comment:
Can this stay `internal` as it's `internal` in SwiftUI? Or, if you have to access it in `TokamakGTK`, maybe let's keep it `public`, but rename to `_PathBox` to highlight that it's not supposed to be re-exported and exposed to users?

```suggestion
  internal class PathBox: Equatable {
```

Reply:
Excellent, I'll rename it.